code/procedures/fieldmaps-to-phase-or-magnitude_fix_all.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# looks into the json-files of fieldmaps.
+for old_json in sub-DTA*/*/fmap/*fieldmap*.json ; do
+	# this line looks complicated, but it just reads the ImageType-array
+	# from the json (example: [ND, ORIGINAL, PRIMARY, P]) and filters it
+	# til there is just a "P" or "M" for phase or magnitude.
```

Comment:
Actually it doesn't. ;-) 
It just looks for that P and simply assumes M otherwise. Better not go for a simple "else", but a proper second condition looking for that M. What if something else accidentally ends up as a file named fieldmap? Better not treat it as if it was a magnitude image, so you can tell afterwards that something's wrong.